Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -950,7 +950,7 @@ static void generate_to_cpp(SourceGenerator& generator, ParameterType& parameter
                 bool may_be_null = !optional_default_value.has_value() || parameter.type->is_nullable() || optional_default_value.value() == "null";
 
                 // Required dictionary members cannot be null.
-                may_be_null &= !member.required;
+                may_be_null &= !member.required && !member.default_value.has_value();
```

Comment:
It would be great if this fix could be split into it's own commit with an explanation of what it is fixing (and while I'm commenting, the trailing comma on WritableStream can be left there)

Reply:
Done.